onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -720,12 +683,40 @@
         // The test cases for Bernoulli op are for informational purpose. The generator operator is
         // non-deterministic and may not produce the same values in different implementations
         // even if a seed is specified.
-        "^test_bernoulli_*"
+        "^test_bernoulli_*",
+        // No support for float8 in numpy
+        "^test_cast_no_saturate*",
```

Comment:
if these are "failing permanently", would it be sufficient to just list them in one place, here?

Reply:
right, this list is added to the previous one, removed

Reply:
removed